source/adapters/cuda/CMakeLists.txt

```diff
@@ -100,7 +101,7 @@ if (UR_ENABLE_TRACING)
     XPTI_ENABLE_INSTRUMENTATION
     XPTI_STATIC_LIBRARY
     )
-  target_include_directories(${TARGET_NAME} PUBLIC
+  target_include_directories(${TARGET_NAME} PRIVATE
```

Comment:
Could you move those unrelated changes to a separate commit?

Reply:
These are related changes. Using compiled UR needs "install" working. And "install" fails unless this wrongly publicly included XPTI is fixed.